React/Views/RCTView.m

```diff
@@ -391,7 +391,7 @@ - (NSString *)accessibilityValue
     }
   }
   NSMutableArray *valueComponents = [NSMutableArray new];
-  NSString *roleDescription = self.accessibilityRoleInternal ? rolesAndStatesDescription[self.accessibilityRoleInternal] : nil; // TODO(OSS Candidate ISS#2710739): renamed prop so it doesn't conflict with -[NSAccessibility accessibilityRole].
+  NSString *roleDescription = self.accessibilityRoleInternal ? rolesAndStatesDescription[self.accessibilityRoleInternal] : nil; // [macOS] renamed prop so it doesn't conflict with -[NSAccessibility accessibilityRole].
```

Comment:
Should the comment be inside the bracket?

Reply:
actually this is inline so no?

Reply:
Good question! I've been going with "no" for this PR, as is also added to documentation. Putting it inside the bracket could also work, but might be awkward here given the comment ends with a bracket.

![Screenshot_20230105-102651.png](https://user-images.githubusercontent.com/6722175/210830742-38001d81-9326-48e8-aeb4-70c5f56c64b9.png)